libdnf/goal/Goal-private.hpp

```diff
@@ -43,6 +43,7 @@ class Goal::Impl {
     std::unique_ptr<PackageSet> protectedPkgs;
     bool protect_running_kernel{true};
     std::unique_ptr<PackageSet> removalOfProtected;
+    Id runningKernelID;
```

Comment:
Why you want to cache running kernel ID? It is already cached by dnf_sack_running_kernel(sack) in sack?

Reply:
I see comment bellow